docs/deploy.md

```diff
@@ -40,6 +42,14 @@ kubectl --namespace azure-databricks-operator-system \
 kubectl apply -f release/config
 ```
 
+## Configure maximum number of run reconcilers
```

Comment:
Please add `MAX_CONCURRENT_RUN_RECONCILES` environment variable in the `config/default/manager_image_patch.yaml` with default value 1.  :)

Reply:
There you go